.github/dockerfiles/cloudserver/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG CLOUDSERVER_IMAGE=ghcr.io/scality/cloudserver:7.70.47
+
+FROM ${CLOUDSERVER_IMAGE}
+
+ADD ./config.json /conf/config.json
+
+RUN apt update && apt install -y curl
```

Comment:
should also cleanup packages after install (delete apt cache etc)

or we could use `node` directly to run healthcheck:
```
test: [ "node", "-e", 'fetch(`http://localhost:9000/health`).then(()=>process.exitCode = 0).catch(()=>process.exitCode = 1)']
```